index.html

```diff
@@ -217,6 +217,41 @@ <h2>Languages</h2>
           </div>
         </div>
       </section>
+      <section class="contact-form">
+        <h3>Contact me</h3>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      <form action="https://formspree.io/f/mayzpyer" method="post">
+        <input
+          class="form"
+          type="text"
+          name="name"
+          placeholder="Name"
+          maxlength="30"
+          required
+        />
+        <input
+          class="form"
+          type="email"
+          name="email"
+          placeholder="Email"
+          maxlength="30"
+          required
+        />
+        <textarea
+          name="message"
+          class="textarea"
+          placeholder="Enter your message here..."
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+        ></textarea>
```

Comment:
- [ ] Good job adding the placeholder 🥇 However, as suggested by the reviewer [here](https://github.com/SolomonTheCoder/Microverse-Portfolio/pull/3#discussion_r1150906049) please consider adding a padding  inside the input element to match the design 💪 See the images below for more clarity 

Your design before | Your design now | Figma design
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/228341226-6ce73734-cedb-4633-8ecb-a215e52c6173.png)| ![image](https://user-images.githubusercontent.com/106119105/228341046-cd0ab5bf-21a0-4cc4-8c0b-2b34505f22f5.png) | ![image](https://user-images.githubusercontent.com/106119105/228341376-b63febcf-1a97-45f8-8258-e2003329730e.png) |

Reply:
- [ ] Still, this issue is not fixed yet, please take the comment of the previous reviewer, you need to fix it before we continue
<img width="211" alt="1" src="https://user-images.githubusercontent.com/77843050/228373595-be108848-2e90-45dc-906d-19cda075fd1d.png">